tests/test_data_client.py

```diff
@@ -26,16 +30,20 @@
 LOCATION_IDS = [LOCATION_ID]
 ORG_ID = "organization_id"
 ORG_IDS = [ORG_ID]
-MIME_TYPES = ["mime_type"]
-START_DATETIME = datetime(2001, 1, 1, 1, 1, 1)
-END_DATETIME = datetime(2001, 1, 1, 1, 1, 1)
+MIME_TYPE = "mime_type"
+MIME_TYPES = [MIME_TYPE]
+URI = "some.robot.uri"
 SECONDS_START = 978310861
 NANOS_START = 0
 SECONDS_END = 978310861
 NANOS_END = 0
+START_TS = Timestamp(seconds=SECONDS_START, nanos=NANOS_START)
+END_TS = Timestamp(seconds=SECONDS_END, nanos=NANOS_END)
+START_DATETIME = START_TS.ToDatetime()
```

Comment:
(flyby) I liked the idea of programmatically showing that `START_DATETIME` and `START_TS` are equivalent.